src/pager/index.js

```diff
@@ -0,0 +1,118 @@
+import React, { Component, PropTypes } from 'react';
+import Icon from '../../infl-components/icon.jsx';
+import classnames from 'classnames';
+
+const MOVING_WINDOW_SIZE = 5;
+const EDGE_WINDOW_SIZE = 2;
+
+export default class Pager extends Component {
+
+  static propTypes = {
+    /**
+     * The current page number (starting at 1).
+     */
+    currentPage: PropTypes.number.isRequired,
+    /**
+     * The number of items that are displayed in each page.
+     */
+    perPage: PropTypes.number,
+    /**
+     * The total number of items that are available.
+     */
+    totalItemCount: PropTypes.number.isRequired,
+    /**
+     * This callback is called when the user navigates to a different page. It
+     * has a single argument which is the new page number.
+     */
+    onPageChangeCallback: PropTypes.func.isRequired,
+    /**
+     * The classname of the page element that is currenting displayed.
+     */
+    selectedClassName: PropTypes.string
+  };
+
+  static defaultProps = {
+    perPage: 10,
+    selectedClassName: ''
+  };
+
+  render() {
+    let {currentPage} = this.props;
+    return (
+      <div className='pt-pager' {...this.props}>
+        <span className='pt-next-page' onClick={this._handleClick.bind(null, currentPage-1)}>
+          <Icon icon='chevron-left'/>
+        </span>
+        {this._getPageNumbers()}
+        <span className='pt-previous-page'  onClick={this._handleClick.bind(null, currentPage+1)}>
+          <Icon icon='chevron-right'/>
+        </span>
+      </div>
+    );
+  }
+
+  _getPageNumbers = () => {
+    let { currentPage } = this.props;
+    let currentPageIndex = currentPage - 1;
+    let totalPageCount = this._getTotalPageCount();
+
+    let pages = Array.from(Array(totalPageCount).keys());
```

Comment:
Array.prototype.from is not supported in all browsers. We can't use it.
(We are not loading babel polyfill in the hub as of yet)